jenkins_pipelines/environments/common/pipeline.groovy

```diff
@@ -94,7 +94,7 @@ def run(params) {
                             break;
                     }
                 }
-                sh "set +x; source /home/jenkins/.credentials set -x; export TF_VAR_CUCUMBER_GITREPO=${params.cucumber_gitrepo}; export TF_VAR_CUCUMBER_BRANCH=${params.cucumber_ref}; export TERRAFORM=${params.terraform_bin}; export TERRAFORM_PLUGINS=${params.terraform_bin_plugins}; ./terracumber-cli ${common_params} --logfile ${resultdirbuild}/sumaform.log ${env.TERRAFORM_INIT} ${env.TERRAFORM_TAINT} --sumaform-backend ${params.sumaform_backend} --runstep provision"
+                sh "set +x; source /home/jenkins/.credentials set -x; export TF_VAR_CUCUMBER_GITREPO=${params.cucumber_gitrepo}; export TF_VAR_CUCUMBER_BRANCH=${params.cucumber_ref}; export TERRAFORM=${params.terraform_bin}; export TERRAFORM_PLUGINS=${params.terraform_bin_plugins}; ./terracumber-cli ${common_params} --logfile ${resultdirbuild}/sumaform.log ${env.TERRAFORM_INIT} ${env.TERRAFORM_TAINT} --sumaform-backend ${params.sumaform_backend} --runstep provision | while read -r line; do echo \"$line\" | sed -E 's/\\[([0-9]{4}-[0-9]{2}-[0-9]{2})T([0-9]{2}:[0-9]{2}:[0-9]{2})\\.[0-9]+Z\\] module\\.([^.]+)\\.module\\.([^.]+)\\.module\\.([^.]+)\\.module\\.[^.]+\\.null_resource\\.[^.]+\\[[0-9]+\\] (.*)/[\\2] \\3.\\4 \\6/'; done"
```

Comment:
Good idea to get rid of all this `module.` that nobody follows, however I would stay with the original timestamp.

Reply:
Agree

Reply:
From [2024-09-30T08:07:09.425Z] to [08:07:09].

- The date is part of all our Jenkins jobs, and this replacement it will only happen in pipelines, so what's the value of adding it for every log line?
- The milliseconds add very little value, if we take into account that the log lines are printed chronologically.

Do you really still prefer to have it complete?

Reply:
Milliseconds maybe not, but `year-month-day` are useful when someone needs a help with debugging on let say master branch from around this `year-month-day-hour`, if you run code next day in the similar hour you might get the same log but on the different codebase if master gets updated on the new commits, it may rise confusions in my opinion, and I do not support removing it.

Reply:
And I won't rely on the jenkins info on the screenshot because this run may disappear any time, but plain text from error log not, attaching this kind of screenshot to bugzilla is not an attractive option indeed.

Reply:
Ok, I will keep year-month-day and filter milliseconds 👍🏼